source/messaging/manage-channels-settings.rst

```diff
@@ -185,16 +185,21 @@ Collapsed Reply Threads (Beta) offers an enhanced experience for users communica
 
 See our `Organizing Conversations using Collapsed Reply Threads (Beta) <https://docs.mattermost.com/messaging/organizing-conversations.html>`__ documentation to learn more about this feature.
 
-Quick reactions to messages
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Click to open threads
+~~~~~~~~~~~~~~~~~~~~~
 
-You can react to messages quickly with your most recently-used emojis by hovering over a message.
+By default, selecting any part of a message opens the reply thread in the right pane. You can disable this behavior. When disabled, reply to a message or select the replies count to open the reply thread in the right pane.
```

Comment:
```suggestion
By default, selecting any part of a message opens the reply thread in the right hand sidebar. You can disable this behavior in **Settings > Display > Click to open threads**. When disabled, reply to a message or select the replies count to open the reply thread.
```

Reply:
I've read this doc and it's unclear as to whether providing the location of the setting is relevant or not. If it's not, feel free to disregard. :)

I changed pane to sidebar but realized I'm not sure if we refer to a pane or a sidebar?